src/devices/AD5328/AD5328.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="samples\**" />

```

Comment:
You can remove the next 3 lines and add `<EnableDefaultItems>false</EnableDefaultItems>` to the PropertyGroup just before.
You will still have to add `<Compile Include="*.cs" />` to this item group

Reply:
done